tests/parity/source_packs/test_v2_non_soak.py

```
@@ -0,0 +1,287 @@
+        "denied",
+        "cancelled",
+        "failed",
+        "recovered",
+    }
+    assert state["resume"] in {"absent", "supplied", "accepted", "rejected"}
+    assert state["billing"] in {"included", "blocked", "unknown", "not_applicable"}
+    assert state["side_effect_count"] == 0
+    assert state["boundary_sha256"] is None
+
+
+def _assert_path(row_id: str, path_name: str, path: dict[str, Any]) -> None:
```

Ruff (0.16.3)

[warning] Unused function argument: `row_id`

(ARG001)